GeneratorLib/Schema.cs

```diff
@@ -70,40 +71,61 @@ public class Schema
         [JsonProperty("gltf_detailedDescription")]
         public string DetailedDescription { get; set; }
 
-        [JsonProperty("additionalProperties")]
-        public Schema DictionaryValueType { get; set; }
+        public Schema AdditionalProperties { get; set; }
 
-        public object Disallowed { get; set; }
+        // TODO implement this for glTF 2.0
+        // Used by Schema.Disallowed
+        // Example: Not : AnyOf : [ required: [a, b], required: [a, c], required: [a, d] ]
+        // TypeReferenceConverter
+        public Schema Not { get; set; }
 
-        public int DivisibleBy { get; set; }
+        public uint? MultipleOf { get; set; }
```

Comment:
Dependencies, DetailedDescription, MultipleOf, OneOf, MaxProperties, MinProperties, UniqueItems, and WebGl are not currently in use by the generator.

MultipleOf is in use by buffer view, so throwing isn't an option.